### ISSUE TITLE
a bug of urllib.parse.urljoin

### ISSUE THREAD

**Issue by tqobqbq** [ArchiveBox/ArchiveBox#1411]
```
from urllib.parse import urljoin
print(urljoin('https://web.archive.org/http://www.example.com/a/b.html',
              'c.html')
      )
#result is 'https://web.archive.org/http:/www.example.com/a/c.html'
```
the change from "http://" to 'http:/' in the sub url is not what I expected.
This bug stem from the filter statement from urljoin:
![image](https://github.com/ArchiveBox/ArchiveBox/assets/49046901/46320463-f423-4dbe-81b3-18913456d33f)
And simpily comment out the filter statement return a proper result in this case.

**Comment by pirate** [comment]
Woah good find, it looks like it's an old known issue in the cpython source:

https://github.com/python/cpython/issues/84774#issuecomment-2076080928

**Comment by pirate** [comment]
Fixed in e5aba0d
